src/wamr/pthread.cpp

```diff
@@ -23,20 +49,55 @@ static int32_t pthread_mutex_init_wrapper(wasm_exec_env_t exec_env,
 
 static int32_t pthread_mutex_lock_wrapper(wasm_exec_env_t exec_env, int32_t a)
 {
-    SPDLOG_DEBUG("S - pthread_mutex_lock {}", a);
+    SPDLOG_TRACE("S - pthread_mutex_lock {}", a);
```

Comment:
Switched `lock` and `unlock` to `TRACE` as they cluttered outputs a lot and, given that they are no-opped, I figured they did not matter that much (for the moment).